app/controllers/concerns/two_factor_authenticatable.rb

```diff
@@ -2,14 +2,18 @@ module TwoFactorAuthenticatable
   extend ActiveSupport::Concern
 
   included do
-    prepend_before_action :authenticate_user!
```

Comment:
According to @monfresh:

> The issue is that the `prepend_before_action :authenticate_user!` callback wasn't working because by default it only applies to non-Devise controllers. To force a user to be authenticated in order to access certain pages that are tied to Devise controllers, we need to pass the `force: true` option.

Source: https://github.com/18F/identity-idp/pull/839#issuecomment-266057923

So when it's just a before_action like this there aren't any arguments, but in this case we need to send them.